tests/func/good/closure_conversion_reduce.catala_en

```diff
@@ -25,14 +25,13 @@ $ catala Typecheck --check-invariants
 
 ```catala-test-inline
 $ catala Lcalc -s S -O --closure-conversion
-let scope S (S_in: S_in {x_in: list of integer}): S {y: integer} =
-  let get x : list of integer = S_in.x_in in
+let scope S (s_in: S_in {x_in: list of integer}): S {y: integer} =
+  let get x : list of integer = s_in.x_in in
   let set y : integer =
     (reduce
-       (λ (x1: (integer, integer)) (x2: (integer, integer)) →
-        if x1.1 < x2.1 then x1 else x2)
+       (closure_y1, to_closure_env ())
        (-1, -1)
-       map (λ (potential_max: integer) → (potential_max, potential_max)) x).0
+       map (closure_y, to_closure_env ()) x).0
```

Comment:
This makes no sense : the argument of `map` cannot be a tuple, has to be a function right ?

Reply:
Closure conversion should not convert closures arguments of `map`, `filter`, etc. As these should be inlined into `for` loops in C.

Reply:
Ok, the above comment should be put as a comment in this test file.